src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -605,7 +648,7 @@ public TimeSpan GetTimeoutOrDefault(TimeSpan? timeout)
         /// <returns></returns>
         public IActorRef ChildActorOf(Props props, string name, CancellationToken cancellationToken = default)
             => ChildActorOfAsync(props, name, cancellationToken)
-                .ConfigureAwait(false).GetAwaiter().GetResult();
+                .Result;
```

Comment:
We still need to use GetAwaiter().GetResult() because this automatically unwraps all AggregateException, using .Result here breaks behavior consistency.